src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
I have found _no other_ `[Alias()]` in the repo. So we should keep consistency and use the same approach - place aliases in ` InitialSessionState.cs`.
Also I set `ReadOnly` because later we can replace it on `None` based on feedback but changing  `None` to `ReadOnly` will be breaking change.